Examples/LambdaFunctions/scripts/build-and-package.sh

```diff
@@ -18,13 +18,22 @@ set -eu
 executable=$1
 workspace="$(pwd)/../.."
 
+echo "-------------------------------------------------------------------------"
+echo "preparing docker build image"
+echo "-------------------------------------------------------------------------"
+docker build . -t builder
+echo "done"
+
 echo "-------------------------------------------------------------------------"
 echo "building \"$executable\" lambda"
 echo "-------------------------------------------------------------------------"
-docker run --rm -v "$workspace":/workspace -w /workspace/Examples/LambdaFunctions builder bash -cl "swift build --product $executable -c release -Xswiftc -g"
+docker run --rm -v "$workspace":/workspace -w /workspace/Examples/LambdaFunctions builder \
+       bash -cl "swift build --product $executable -c release"
```

Comment:
Just to fill a knowledge gap on my site: What's the benefit of using `bash -cl "cmd"` vs. just `cmd`

Reply:
old habit to make sure we get login shell: https://www.gnu.org/software/bash/manual/html_node/Invoking-Bash.html